src/js/pagination/paging-collection.js

```diff
@@ -343,11 +345,9 @@
                     value = value.join(',');
                 }
                 this.filterableFields[fieldName].value = value;
-                if (_.isUndefined(this.queryParams.fieldName)) {
```

Comment:
I'm not totally clear on why the conditional is being removed here -- does something about the refactor make it unnecessary? Was it ever needed?